pkg/controller/sync/dispatch/managed.go

```diff
@@ -45,21 +45,20 @@ type FederatedResourceForDispatch interface {
 // ManagedDispatcher dispatches operations to member clusters for resources
 // managed by a federated resource.
 type ManagedDispatcher interface {
-	OperationDispatcher
+	UnmanagedDispatcher
```

Comment:
There's probably a good reason to introduce this indirection, but it's difficult to contextualize the need for it based solely on this review.

Reply:
If you need context, please look at subsequent PRs for #612